### DISCUSSION TITLE
连接数据库后出现 “is null safe eq join is not supported” 的问题

### DISCUSSION THREAD

**Discussion by a123wyn** [apache/doris#6715] [A - General / Q&A]
新手小白~~
打算用tableau连接doris来进行可视化，mysql是5.3的版本，想问下大家连接doris里的数据表展示时有没有出现这个问题~~应该如何解决哇？

**Comment by hf200012** [comment]
Post your SQL statement

**Reply by a123wyn** [reply]
您好感谢回复~~计算字段有点多我不清楚是哪个出了问题，但是都没有join语句，都是一些count和sum的聚合函数，请问这种报错一般是哪里出现问题呢？

**Reply by a123wyn** [reply]
有计算今日数据的例如：
COUNTD(IF date([Prc Time]) = TODAY()
then [Store Id]
ELSE null
end)
还有计算周环比和日环比的

**Reply by a123wyn** [reply]
我觉得自己好像找到问题所在了，可能是tableau中支持的fixed功能无法兼容doris的原因~感谢~~
